bench/workgen/workgen.cxx

```diff
@@ -861,13 +856,16 @@ ThreadRunner::op_create_all(Operation *op, size_t &keysize, size_t &valuesize)
 
             // We are single threaded in this function, so do not have
```

Comment:
I think this comment should be removed now we are allowing the creation of tables on the fly?

Reply:
Remv

Reply:
Done in c3c11339810d3ba82ae1092abc53c3a1a7edf331